Classes/Core/Testbase.php

```diff
@@ -249,9 +271,27 @@ public function setUpInstanceCoreLinks($instancePath): void
      * @param non-empty-string[] $extensionPaths Contains paths to extensions relative to document root
      * @throws Exception
      */
-    public function linkTestExtensionsToInstance($instancePath, array $extensionPaths): void
+    public function linkTestExtensionsToInstance(string $instancePath, array $extensionPaths): void
     {
         foreach ($extensionPaths as $extensionPath) {
+            $packageInfo = $this->composerPackageManager->getPackageInfo($extensionPath);
+            if ($packageInfo instanceof PackageInfo) {
+                $installPath = $packageInfo->getRealPath() . '/';
+                $destinationPath = $instancePath . '/typo3conf/ext/' . $packageInfo->getExtensionKey();
+                // @todo Need we some "relative" or "short-path" calculation for the symlink to avoid absolute source ?
+                $success = symlink($installPath, $destinationPath);
```

Comment:
```suggestion
                $success = @symlink($installPath, $destinationPath);
```

Reply:
I thought I used the suppress for all new places after your recenently merged patch. Will check this again. Thanks.

Reply:
Missed it in the backport. Fixed.